app/views/teams/_form.html.erb

```diff
@@ -4,7 +4,7 @@
   <%= render partial: "application/validation_errors", locals: {model: @team} %>
 
   <div class="form-group">
-    <%= f.label t("helpers.name"), :class => 'control-label col-lg-2' %>
+    <%= f.label t("helpers.name"), :class => 'control-label col-lg-2 required', :required => true %>
```

Comment:
Very nice! :-)

Reply:
<img width="1440" alt="screen shot 2018-01-15 at 14 11 57" src="https://user-images.githubusercontent.com/18228544/34944004-30c0ff78-f9fe-11e7-9330-d5d7e47bb1ec.png">

I can't see any required indicators at the moment. Please check! Maybe after merging 'dev' into current branch, something went wrong?